Octokit.Tests.Conventions/PaginationTests.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Xunit;
+
+namespace Octokit.Tests.Conventions
+{
+    public class PaginationTests
+    {
+        [Theory(Skip = "Disable this to run it and find all the places where things break")]
```

Comment:
If I get a :thumbsup: and land this PR, I can then run this test and extract tasks to get things update accordingly.

For now, it's not so noisy...